src/iocore/cache/CacheVC.cc

```diff
@@ -775,7 +775,9 @@ CacheVC::scanObject(int /* event ATS_UNUSED */, Event * /* e ATS_UNUSED */)
         goto Lskip;
       }
       if (!hostinfo_copied) {
-        memccpy(hname, vector.get(i)->request_get()->host_get(&hlen), 0, 500);
+        auto host{vector.get(i)->request_get()->host_get()};
+        hlen = static_cast<int>(host.length());
+        memccpy(hname, host.data(), 0, 500);
```

Comment:
This is concerning here since string_view.data() isn't necessarily null terminated, but this copy is looking for one.  This should be memcpy or strncpy (but still add the termination).  Ideally hname would also be a string_view, but I'd have to dig further to see if that is possible without more refactoring.